docs/ens.md

```diff
@@ -0,0 +1,104 @@
+---
+id: ens
+title: Adding accounts to an ENS domain
+sidebar_label: Using ENS with DOT/KSM accounts
+---
+
+ENS (Ethereum Name Service) is a system of smart contracts on the Ethereum blockchain which allows
+users to claim domain names like `bruno.eth`. Supporting wallets can then allow senders to input ENS
+domains instead of long and unwieldy addresses. This prevents phishing, fraud, typos, and adds a
+layer of usability on top of the regular wallet user experience.
+
+_Note: You will need an ENS name and an Ethereum account with some ether in it to follow along with
+this guide. To register an ENS name, visit the [ENS App](https://app.ens.domains) or any number of
+subdomain registrars like [Nameth](https://nameth.io). You will also need some way to use the
+Ethereum address - we recommend [Frame](https://frame.sh/) or [Metamask](https://metamask.io)._
```

Comment:
Usually we put notes in quote blocks:

```
> NOTE: You will...
```

Reply:
A quotenote seemed a little aggressive for such a long paragraph, but for the sake of standardizing, I agree